core/controllers/outgoing_emails.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import email_manager
+import feconf
+import webapp2
+
+
+class ThankYouMailHandler(webapp2.RequestHandler):
```

Comment:
This should take the email params from the frontend and pass them to email_manager which sends a mail to the admin.

Reply:
So I am not sure about this comment. Am I supposed to override the handler and create a custom baseHandler like in Oppia's main app? 

Otherwise, I rename the handler, make the handler read the JSON data and pass it forward to the email manager.

Reply:
Yep, create a basehandler and then make a subclass of that handler to serve for this particular URL, like in Oppia's core/controllers/base.py. That handler should read the JSON data that's incoming and then pass it on to the email manager.